pkg/volume/glusterfs/glusterfs.go

```diff
@@ -394,7 +395,7 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
 	// Refer to backup-volfile-servers @ http://docs.gluster.org/en/latest/Administrator%20Guide/Setting%20Up%20Clients/
 
 	if (len(addrlist) > 0) && (addrlist[0] != "") {
-		ip := addrlist[0]
+		ip := addrlist[rand.Intn(len(addrlist))]
```

Comment:
How does it interact with `backup-volfile-servers`? The comment above suggests that it does not really matter which address is used as mount source.

Reply:
@jsafrane  True.. The effect on `backup-volfile-servers` is negligible,  the reason being `backup-volfile-servers` used as  a list/iteration to try out if one/attempted mount server is not available.  Without randomization most of the mount happens on single/same server ( as its available mostly) and this patch should avoid such huge number of mounts on the same server.